README.md

```diff
@@ -1,2 +1,81 @@
-# resources
-A curated list of Angular resources
+# Resources
+
+A crowd-sourced curated list of Angular resources.
+
+## Table of Contents
+
+- [Official Angular resources](#official-angular-resources)
+- [Beginners Journey](#beginners-journey)
+- [Angular](#angular)
+  - [Angular Blogs/Articles](#angular-blogs)
+  - [Angular Books](#angular-books)
+  - [Angular Courses](#angular-courses)
+  - [Angular Example Applications](#angular-example-applications)
+  - [Angular Videos](#angular-videos)
+- [TypeScript](#typescript)
+  - [TypeScript Blogs/Articles](#ts-blogs)
+  - [TypeScript Books](#ts-books)
+  - [TypeScript Courses](#ts-courses)
+  - [TypeScript Videos](#ts-videos)
+- [RxJS](#rxjs)
+  - [RxJS Blogs/Articles](#rxjs-blogs)
+  - [RxJS Books](#rxjs-books)
+  - [RxJS Courses](#rxjs-courses)
+  - [RxJS Videos](#rxjs-videos)
+- [Libraries](#libraries)
+  - [NgRx](#ngrx)
+
+### Official Angular resources
+
+- [Angular Website.](https://angular.io/) - The web development framework for building the future.
+- [Angular Blog](https://blog.angular.io/) - News and Tips from the Angular Team.
+- [Angular Github](https://github.com/angular) - Angular is developed in public, take a look at the Angular repositories.
+- [Angular Twitter](https://twitter.com/angular) - Follow @angular on twitter.
+- [Angular Youtube](https://www.youtube.com/@Angular) - Videos and Shorts from the Angular Team.
+
+### Beginners Journey
+
+### Angular
+
+<h5 id="angular-blogs">Blogs/Articles</h5>
```

Comment:
Looking at how this looks on GH, the h5 (.875em) seems to small compared to the normal text. Probably better to make it h4(1em or 16px). Thoughts?